### PR TITLE
lerna: add page

### LINKED ISSUE #6325: page request: lerna
A tool for managing JavaScript projects with multiple packages: https://github.com/lerna/lerna / https://lerna.js.org

### PR DESCRIPTION
**Pull request by pixelcmtd**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

closes #6325

**Version of the command being documented (if known):**
4.0.0

### CODE REVIEW

File: pages/common/lerna.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# lerna
+
+> A tool for managing JavaScript projects with multiple packages.
+> More information: <https://lerna.js.org/>.
```
Comment (navarroaxel) [inline pages/common/lerna.md:4]:
```suggestion
> More information: <https://lerna.js.org>.
```

File: pages/common/lerna.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Initialize project files (`lerna.json`, `package.json`, `.git`, …):
```
Comment (navarroaxel) [inline pages/common/lerna.md:6]: Don't use special chars like `…` could be not compatible with the user's terminal.
```suggestion
- Initialize project files (`lerna.json`, `package.json`, `.git`, .etc):
```

File: pages/common/lerna.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`lerna init`
+
+- Install all external dependencies of each package and symlink together local dependencies:
+
+`lerna bootstrap`
+
+- Run `npm run {{script}}` for every package that contains that `{{script}}`:
```
Comment (navarroaxel) [inline pages/common/lerna.md:14]: Perhaps this better
```suggestion
- Run a specific script for every package that contains that script in the `package.json`:
```

File: pages/common/lerna.md (hunk 2 above)
Comment (bl-ue) [reply pages/common/lerna.md:6]: @navarroaxel right, but it should be `etc.` not `.etc`:
```suggestion
- Initialize project files (`lerna.json`, `package.json`, `.git`, etc.):
```

Comment (pixelcmtd) [reply pages/common/lerna.md:6]: Why not `...`?

Comment (navarroaxel) [reply pages/common/lerna.md:6]: because `...` is for a pattern, like the sequence `1, 2, ...`

**Review by navarroaxel — APPROVED** [review]

**Review by einverne — APPROVED** [review]
Looks good.

**Review by mfrw — APPROVED** [review]
Thank you @pixelcmtd for your contribution.
LGTM :)

**Pull request merged**